Snakefile

```diff
@@ -1,3 +1,9 @@
+import os
+from snakemake.remote.S3 import RemoteProvider as S3RemoteProvider
+S3 = S3RemoteProvider(access_key_id=os.environ["AWS_ACCESS_KEY_ID"],
+                      secret_access_key=os.environ["AWS_SECRET_ACCESS_KEY"])
```

Comment:
You can just use `S3RemoteProvider()` with no args and it will pick up those environment variables automatically (as well as supporting file-based config and things like `AWS_PROFILE`). This is more consistent with all other AWS tools (and the ones we've written for ID3C).

Reply:
Interesting. Do the environment variables have to be specifically named `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` in order for the automatic detection to work?

Reply:
Yes.